ctapipe/tools/dl1_merge.py

```diff
@@ -34,15 +34,15 @@
                   '/simulation/event/telescope/parameters',
                   '/simulation/event/telescope/images',
                   '/simulation/service/shower_distribution']
+optional_nodes = ['/simulation/service/shower_distribution',
+                  '/simulation/event/telescope/images',
```

Comment:
This is duplicated to the below elements in `allow_list` simu. Is that needed?

Reply:
I need the `allowlist_simu` for checking the file. When its real data and doesn't have the `simulation`-group, it should not give out a warning (so i can skip these groups in checking). `/simulation/event/subarray/shower` is also just in `allowlist_simu`. And i was thinking that its more clearly this way and you can just add groups for future purpose.